### ISSUE TITLE
Can't add computed fields as advertised in the documentation

### ISSUE THREAD

**Issue by zadigus** [hasura/graphql-engine#3727]
I would like to add a computed field of the very same kind as that presented in the [documentation](https://docs.hasura.io/1.0/graphql/manual/schema/computed-fields.html#table-computed-fields). I am working with hasura v1.0.0. According to this [documentation](https://docs.hasura.io/1.0/graphql/manual/schema/computed-fields.html#add-computed-field), there should be a "Computed fields" section in my table's "Modify" tab, which I don't see. Do I need to install something special? Should the table satisfy some condition to make it happen?

**Comment by rikinsk** [comment]
@zadigus Console support for computed fields is going to be released in `v1.1.0`. It is currently available in the `v1.1.0-beta.2` release if you would like to use it. A stable `v1.1.0` will be released very soon.

**Comment by zadigus** [comment]
How do I do it without console support? I tried to apply the following migration:
```
- args:
    sql: |
      CREATE FUNCTION users_full_name(users_row users)
      RETURNS TEXT AS $$
        SELECT users_row.first_name || ' ' || users_row.last_name
      $$ LANGUAGE sql STABLE;
  type: run_sql
```
but then I see no new computed field in my graphql query (in the console). What should I do that see my new `full_name` computed field? Do I need to make it happen with metadata then? I think I saw something like that in the documentation.

**Comment by zadigus** [comment]
I'm asking how to do it with the metadata, because I can't open the console with the latest docker images `v1.1.0-beta.2`. If my graphql-engine has that version, then a
```
hasura console
```
results in
```
?[31mFATA?[0m[0002] version check: [cli: v1.0.0] [server: v1.1.0-beta.2] versions incompatible: cli version (major.minor) should be equal or ahead of server version, please update cli
```
Is there another way to open the console? because
```
hasura update-cli
```
results in

![image](https://user-images.githubusercontent.com/8761254/72598579-5a920380-3910-11ea-9b22-28b25b9d2766.png)

**Comment by rikinsk** [comment]
@zadigus The CLI version check issue is a bug which is being worked on. In the meanwhile you can use the metadata api to add your computed field as documented [here](https://docs.hasura.io/1.0/graphql/manual/api-reference/schema-metadata-api/computed-field.html)

**Comment by zadigus** [comment]
Ok. The feature is not urgent for me. I'll have a look into the link you provide. For me, it's fine like that.

**Comment by OLDIN** [comment]
Has the bug been fixed? I need to export metadata.

**Comment by rikinsk** [comment]
@OLDIN This bug has been fixed and will be released with v1.1.0. In the meanwhile you can use the CLI build of the [v1.1.0-beta3 release](https://github.com/hasura/graphql-engine/releases/tag/v1.1.0-beta.3) (available under `Assets`) to do the export
